build/build.sh

```diff
@@ -15,8 +15,17 @@ yum install -y cpio yum-utils zip
 # extract binaries for clamav, json-c, pcre
 mkdir -p /tmp/build
 pushd /tmp/build
-yumdownloader -x \*i686 --archlist=x86_64 clamav clamav-lib clamav-update json-c pcre2 libxml2 bzip2-libs libtool-ltdl xz-libs
+
+# Download the clamav package that includes unrar
+curl -L --output clamav-0.102.4-14050.el7.art.x86_64.rpm http://www6.atomicorp.com/channels/atomic/centos/7/x86_64/RPMS/clamav-0.102.4-14050.el7.art.x86_64.rpm
 rpm2cpio clamav-0*.rpm | cpio -vimd
+
+# Download libcrypt.so.1
+curl -L --output glibc-2.17-307.el7.1.x86_64.rpm http://mirror.centos.org/centos/7/os/x86_64/Packages/glibc-2.17-307.el7.1.x86_64.rpm
```

Comment:
```suggestion
curl -L --output glibc-2.17-307.el7.1.x86_64.rpm https://mirror.centos.org/centos/7/os/x86_64/Packages/glibc-2.17-307.el7.1.x86_64.rpm
```

Reply:
I haven't found this package available with https